### DISCUSSION TITLE
Failed to connect to radarr

### DISCUSSION THREAD

**Discussion by yuval-ziv** [sct/overseerr#2309] [Help]
Hello,
I've been trying to setup overseerr. 
I already have radarr running well on my pc.
It's running with a self signed cert, but also with http on different port.
I have Forms authentication in Radarr.

When testing radarr (add service), I keep on getting failed to connect.
I've tried both ports with/without ssl, checking my api key multiple times, and even removing the forms authentication from radarr, but still no luck.

Can anyone help me?

**Comment by yuval-ziv** [comment]
Everything is running on the same windows 10 desktop.
Overseerr runs on docker desktop while radarr runs natively.

I get this error in overseerr container logs:
`[error][Radarr]: [Radarr] Failed to retrieve system status: connect ECONNREFUSED 127.0.0.1:8787`

**Reply by mikecodre** [reply]
I have the same issue, did you solve it?

**Comment by mikecodre** [comment]
The issue for me was solved by using host.docker.internal as hostname!!!

**Comment by PH1L900D** [comment]
Took me far to long to figure this out but try:

docker run -d \
  --name overseerr \
  -e LOG_LEVEL=debug \
  -e TZ=Pacific/Auckland \
  -p 5055:5055 \
  -v /home/phil/Overseerr:/app/config \
  --restart unless-stopped \
  --network="host" \
  sctx/overseerr

Note the --network="host" arg. See https://stackoverflow.com/questions/24319662/from-inside-of-a-docker-container-how-do-i-connect-to-the-localhost-of-the-mach#:~:text=Ok%2C%20your%20localhost%20server%20has,%2D%2Dnet%3D%22bridge%22%20.

**Reply by mikecodre** [reply]
Unfortunately I'm on windows so it's not supported... (the network host argument)
I need to find a workaround though because I can't access overseerr from anywhere other than the pc it's running on.
